source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -393,27 +507,37 @@ Confirm that the telemetrics header file is located on the system at :file:`usr/
 
    Severity:
     | Type: uint32_t
-    | Value:  Severity field value. Accepted values are in the range 1-4, with 1 being the lowest severity, and 4 being the highest severity. Values provided outside of this range are clamped to 1 or 4. [low, med, high, crit]
+    | Value:  Severity field value. Accepted values are in the range 1-4, with
```

Comment:
Please remove the single `|`, or pipe, character. It's non-standard, and it causes build-errors. Reformat these sections. For these headers, like Severity, please use **Severity** and check each one of the header's left-alignment to assure it aligns with the previous paragraph above, or instruction--and not the code-block. I can walk you through the fussier parts of this if you like.

Reply:
I experimented with correcting the `::` and removing the `|` characters to achieve a clean build. Just takes a little work.

Reply:
Removed pipes and left-aligned subsequent text to header left margin.